README.md

```diff
@@ -244,6 +244,13 @@ Refer to the `reth` configuration [documentation](https://reth.rs/cli/reth/node.
 
 Navigate to your `op-node` directory and start service by running the command:
 
+**Note**:
+
+- Please make sure to patch your `op-node` with [`lisk-hotfix.patch`](./geth/lisk-hotfix.patch) for an unhandled `SystemConfig` event emitted, affecting the Lisk nodes resulting in error logs. This patch is temporary until our RaaS provider updates the `SystemConfig` contract.
```

Comment:
The patch is only needed when running Lisk Sepolia nodes. May be we can update it to
```
Please make sure to patch your `op-node` with [`lisk-hotfix.patch`](./geth/lisk-hotfix.patch) for an unhandled `SystemConfig` event emitted, affecting the Lisk Sepolia node resulting in error logs.
´´´

Reply:
This affects both Mainnet and Sepolia Testnet. Let's discuss this offline.